app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java

```diff
@@ -39,7 +41,8 @@
 
 import dagger.Lazy;
 
-public class MainActivity extends BaseActivity<IMainPresenter> implements NavigationView.OnNavigationItemSelectedListener, IMainView {
+public class MainActivity extends BaseActivity<IMainPresenter> implements
```

Comment:
This is a recipe to create God Object. About Fragment should be responsible for handling the action bar changes